keylime/common/retry.py

```diff
@@ -0,0 +1,11 @@
+"""
+SPDX-License-Identifier: Apache-2.0
+Copyright 2021 Angelo Ruocco - IBM Research Lab Zurich
+"""
+
+def retry_time(exponential, base, ntries):
+    if base < 1 and exponential:
+        print("Base value %f incompatible with exponantial backoff" % base)
```

Comment:
Instead of using print, pass the logger as an argument.

Can you add a test for this function?